libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/InputDialog.cs

```diff
@@ -266,7 +266,7 @@ public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext d
                     var (value, error) = this.DefaultValue.TryGetValue(dc.State);
                     if (this.DefaultValueResponse != null)
                     {
-                        var response = await this.DefaultValueResponse.BindAsync(dc, cancellationToken).ConfigureAwait(false);
+                        var response = await this.DefaultValueResponse.BindAsync(dc, cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
https://github.com/microsoft/botbuilder-dotnet/blob/4677d4ec6c5bd3659ec834d4c175f3036208e363/libraries/Microsoft.Bot.Builder.Dialogs/Prompts/ActivityPrompt.cs#L65

see here, can we defense explicitly on cancellationToken in BindAsync

Reply:
added explicit type checking for passing cancellationToken as data in BindAsync